src/Query/DefaultQueryFactory.php

```diff
@@ -18,46 +19,66 @@ public function __construct(array $state)
 
     public function create(ExportConfig $exportConfig, DbConnection $connection): string
     {
-        $sql = [];
+        $sql = array_merge(
+            iterator_to_array($this->createSelect($exportConfig, $connection)),
```

Comment:
Z praxe vysla poziadavka, ze niekedy potrebuje programator upravit iba cast SQLka, tak som to rozdelil do viacerych metod. Tym ze sa pouzili generatory, tak nemusi vyt v kazdej metode na zaciatku `$sql = []` a na konci `return $sql;`